src/Control/Monad/Aff.js

```diff
@@ -96,6 +96,61 @@ exports._forkAff = function (nonCanceler, aff) {
   };
 }
 
+exports._forkAll = function (nonCanceler, foldl, affs) {
+  var voidF = function(){};
+
+  return function(success, error) {
+    var cancelers = foldl(function(acc) {
+      return function(aff) {
+        acc.push(aff(voidF, voidF));
+        return acc;
+      }
+    })([])(affs);
```

Comment:
Nah, I can see it's fine, it's just the mutating of the array in the fold while constructing it that made me look twice. :smile: